src/assets/styles/_variables.scss

```
@@ -1 +1,8 @@
 $primary-color: #6372F1;
+$p-light-grey: rgba(0,0,0,0.62);
```

please use variables
```suggestion
$light-grey: rgba($black, 0.62);
```

---

resolved....................Thank you